docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -272,41 +275,113 @@ Where,
 
 We can refer to the outcome rate multiplied by (1 - PAF) as the "risk-deleted outcome rate."
 
-.. todo::
+Using RRs and PAFs from GBD
+---------------------------
 
-    Add a note about bias this introduces...
+While GBD reports RRs and PAFs, they are often not suitable for use in Vivarium models.
+This is because we typically model GBD causes as dynamic state-transition models,
+and are interested in applying RRs and PAFs to *transitions* in the model.
```

Comment:
Maybe at the end of the last sentence, add something like, "while GBD's RRs and PAFs are applied to static, population level quantities, namely YLLs and YLDs."

I know that overlaps a bit with the next paragraph, so maybe edit a bit, but I think it's worth pointing out _why_ applying the RRs and PAFs to our dynamic state-transition models is incompatible with GBD.